routers/routes/routes.go

```diff
@@ -523,6 +523,11 @@ func RegisterRoutes(m *macaron.Macaron) {
 				return
 			}
 		})
+		m.Group("/branches", func() {
```

Comment:
@lunny I think it should loop through tabs and then check if user has access. I can create PR for that?